packages/core/src/components/Loader/Loader.types.ts

```diff
@@ -0,0 +1,19 @@
+export type LoaderColors = "primary" | "secondary" | "onPrimary" | "dark";
+
+export type LoaderSizes = "xs" | "small" | "medium" | "large";
```

Comment:
Is it used anywhere else? Do we actually need to export it?
It might be confusing if we export both

Reply:
@talkor I think for now we should just export it as it is used also for the const in the component and when we combine all the size types we will anyhow get rid of it. wdyt?

Reply:
NP